pkg/param/param_test.go

```diff
@@ -359,6 +359,7 @@ func (s *ParamsSuite) testNewTemplateParams(ctx context.Context, c *C, dynCli dy
 	c.Assert(err, IsNil)
 	c.Assert(cm, NotNil)
 
+	contxt := context.TODO()
```

Comment:
@viveksinghggits here `ctx` is already initialized as a parameter.

Reply:
Use `ctx`
```suggestion
```